interview-questions.md

```diff
@@ -8,32 +8,32 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Compare and contrast arrays and hashes in Ruby. Include at least two differences and two similarities.
 
-Your answer:
+Your answer: Both can hold multiple pieces of data and can itterate through them. Hashes can hold key-value pairs and objects/symbols. While Hases cannot. 
 
-Researched answer:
+Researched answer: Both arrays and hashes are used to store collections of data. They can store multiple values or items in a single variable, bot can also be iterated over using similar methods, such as each, map, and select. Arrays are ordered collections of elements, and each element in an array is assigned an index, starting from 0. Hashes are not ordered, which means the elements in a hash have no guaranteed order using keys instead of indexes.
 
 2. What does it mean to implement a TTD workflow?
 
-Your answer:
+Your answer: To implement a TTD workflow means to test everything for a 'good' fail first, then build the code or function to  make the test pass.
 
-Researched answer:
+Researched answer: To implement a TTD workflow means to write a Test (Red phase) to describe the expected behavior of the code. Then write the minimum code to make the test pass (Green phase). Then refactor the code (Refactor phase), to make it as clean and efficent as possible while still passing the test!
 
 3. Why would super be used in a Ruby class?
 
-Your answer:
+Your answer: To call back to a method used in the original parent class (inception).
```

Comment:
I would recommend expanding this answer in an interview.  Interviewers want to get an idea on the concepts that you know along with how you communicate them, so elaborating whenever possible is extremely valuable.